src/nyc_trees/apps/home/templates/home/partials/location_search.html

```diff
@@ -0,0 +1,19 @@
+<script type="text/template" id="template-map-search-toolbar-button">
+    <a class="icon-search location-search-show-button" href="javascript:;" title="Search"></a>
+</script>
+
+<script type="text/template" id="template-map-search">
+    <div class="location-search-container" style="background: pink; display: none;">
```

Comment:
The explicit ``pink`` background is here as a placeholder until styling is applied.